docs/src/components/ComponentDoc/ComponentExample/ComponentExampleColorPicker.tsx

```diff
@@ -0,0 +1,76 @@
+import { ProviderContextPrepared } from '@stardust-ui/react'
+// @ts-ignore
+import { ThemeContext } from '@stardust-ui/react-fela'
+import * as Color from 'color'
+import * as _ from 'lodash'
+import * as React from 'react'
+
+import { ColorVariants } from 'src/themes/types'
+
+type ComponentExampleColorPickerProps = {
+  onChange: (colorValue: string) => void
+  variableValue: string
+}
+
+const ComponentExampleColorPicker: React.FunctionComponent<
+  ComponentExampleColorPickerProps
+> = props => {
+  const { onChange, variableValue } = props
+  const { theme } = React.useContext<ProviderContextPrepared>(ThemeContext)
+
+  const handleClick = (colorValue: string) => () => {
+    onChange(colorValue)
+  }
+  // Some colors are strings only, i.e. black and white
+  const filteredColors = _.pickBy(theme.siteVariables.colors, _.isPlainObject)
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        lineHeight: '2rem',
+        fontSize: '10px',
+        fontFamily: 'monospace',
+        backgroundImage:
+          'url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAKUlEQVQoU2NkYGAwZkAD////RxdiYBwKCv///4/hGUZGkNNRAeMQUAgAtxof+nLDzyUAAAAASUVORK5CYII=")',
+        backgroundRepeat: 'repeat',
+      }}
+    >
+      {_.map(filteredColors, (colorShades: ColorVariants, colorName: string) => (
+        <div style={{ display: 'flex' }}>
+          <strong style={{ display: 'block', width: '12ch', background: '#fff' }}>
+            {_.startCase(colorName)}
+          </strong>
+
+          {_.map(colorShades, (shadeValue, shadeName) => {
+            const isActive = variableValue === shadeValue
+            const contrastColor = Color(shadeValue).isDark() ? '#fff' : '#000'
+
+            return (
+              <div
```

Comment:
Can we have a title here with the actual color value?

Reply:
![image](https://user-images.githubusercontent.com/14183168/62782104-d2cee000-bab9-11e9-93c3-309053e01f14.png)

Added 👍